src/api-endpoints.ts

```diff
@@ -5210,6 +5218,7 @@ type LanguageRequest =
   | "mathematica"
   | "mermaid"
   | "nix"
+  | "notion formula"
```

Comment:
why was this added?

Reply:
Just an artifact of someone adding a new language but not publishing the `sdk` 😢